packages/ui/src/kit/VInput/VInput.vue

```diff
@@ -19,12 +22,27 @@
       v-bind="$attrs"
       v-on="$listeners"
     >
+      <icon-atom
+        v-if="isLoading"
```

Comment:
soo, if i pass `error` and `is-loading` and `is-resetable` props, i will see three icons?
maybe use one icon-atom and svg-atom for different cases and set correct icon by props?
![image](https://user-images.githubusercontent.com/756731/78412865-aace5700-761d-11ea-99f3-cf71766442df.png)

loader icon also have not center spin, but have some bias for 1-2 px. 
you can see that loader in buttons doesn't have any bias

Reply:
well, we can show always only one icon, but I guess we at least should showng error and reset icons together, because when input errored, we should be able to clear it

loader can be showed instead of error icon

@kazurus ?

Reply:
there are no any bias as well, all looks great 👍 
![image](https://user-images.githubusercontent.com/13561530/78421632-d115e600-7661-11ea-9e02-3eedf5287c46.png)

Reply:
as I can see, we are trying to combine an elephant and a dog. Maybe we need separate component VSearch, for example. What do you think?

Reply:
@kazurus take this opportunity to ask - what's the difference between v-naming and common names? do you use different naming convention in specific cases or this is random? for example, in Vuetify all components have v-prefix

> Maybe we need separate component

Do you want reject this pr and add all in vault? We already have Vsearch there

Reply:
In general, see no problems with loading, reset and error in one input. For example, Vuetify text field with loading, reset and counter :)
![image](https://user-images.githubusercontent.com/13561530/78423773-67053d00-7671-11ea-8bb9-54b0ede26eb0.png)

But, we can stay v-input simple and add all extra-features in vault only.

Reply:
1. In vuetify lib all features are consistent and can be applied in same time
2. V prefix - we use for controls without domain specific, but in vault it's not strict, as I remember
3. Maybe we should have VSearchInput

Reply:
3 - have sense as well